beacon-chain/sync/service_test.go

```diff
@@ -0,0 +1,71 @@
+package sync
+
+import (
+	"context"
+	"testing"
+	"errors"
+
+	"github.com/prysmaticlabs/prysm/beacon-chain/internal"
+)
+
+
+func setupTestSyncService(t *testing.T) *Service {
+	db := internal.SetupDB(t)
+	defer internal.TeardownDB(t, db)
+
+	if err := db.InitializeState(); err != nil {
+		t.Fatalf("Failed to initialize state: %v", err)
+	}
+
+	cfg := &Config{
+		ChainService:  &mockChainService{},
+		P2P:           &mockP2P{},
+		BeaconDB:      db,
+		AttestService: &mockAttestService{},
+	}
+
+	return NewSyncService(context.Background(), cfg)
+}
+
+
+func TestStatusQuerierError(t *testing.T) {
+    ss := setupTestSyncService(t)
+	_, querierErr := ss.Querier.IsSynced()
+
+	if querierErr != nil {
```

Comment:
If this is nil, you aren't testing anything. How can you be certain that this test is testing the error if there is a path that allows nothing to be tested?